packages/app-builder-lib/src/targets/nsis/NsisTarget.ts

```diff
@@ -73,8 +86,16 @@ export class NsisTarget extends Target {
     NsisTargetOptions.resolve(this.options)
   }
 
+  protected buildUniversalInstaller() {
+    const buildSeparateInstallers = this.options.buildUniversalInstaller === false
+    return !buildSeparateInstallers
+  }
```

Comment:
``` javascript
protected buildUniversalInstaller() {
    return this.options.buildUniversalInstaller 
 }
```

Reply:
`buildUniversalInstaller` is optional, so we must check against explicit false.

Reply:
This is not a get function, so there shouldn't be any caching, right?

Reply:
Do we need to cache the result? It's just reading from options.

But I agree, it could simply be a get function. I'll update the PR